_unittest/test_01_configuration_files.py

```diff
@@ -184,13 +184,12 @@ def test_04a_icepak(self, icepak_a, aedtapp, add_app):
             "test_dataset",
             [1, 2, 3, 4],
             [1, 2, 3, 4],
-            zlist=None,
-            vlist=None,
+            z_list=None,
+            v_list=None,
```

Comment:
only "y"

Reply:
@maxcapodi78 I can't use only "y" or "z" because the method requires also y_list and y_units. Same for the other comment below